addons/mail/static/src/components/channel_member/channel_member.xml

```diff
@@ -2,23 +2,23 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ChannelMember" owl="1">
-        <div class="o_ChannelMember d-flex align-items-center mx-3 my-1" t-attf-class="{{ className }}" t-att-data-partner-local-id="member.localId" t-ref="root">
-            <div class="o_ChannelMember_avatarContainer position-relative flex-shrink-0 o_cursor_pointer" t-on-click="() => channel.onClickMemberAvatar(member)">
-                <img class="o_ChannelMember_avatar rounded-circle w-100 h-100" t-attf-src="/mail/channel/{{ channel.id }}/partner/{{ member.id }}/avatar_128" alt="Avatar"/>
+        <div class="o_ChannelMember d-flex align-items-center mx-3 my-1" t-attf-class="{{ className }}" t-att-data-partner-local-id="channelMember.partner.localId" t-ref="root">
+            <div class="o_ChannelMember_avatarContainer position-relative flex-shrink-0 o_cursor_pointer" t-on-click="channelMember.onClickAvatar">
+                <img class="o_ChannelMember_avatar rounded-circle w-100 h-100" t-attf-src="/mail/channel/{{ channelMember.channel.id }}/partner/{{ channelMember.partner.id }}/avatar_128" alt="Avatar"/>
 
-                <t t-if="member.im_status and member.im_status !== 'im_partner'">
+                <t t-if="channelMember.partner.im_status and channelMember.partner.im_status !== 'im_partner'">
                     <PartnerImStatusIcon
                         className="'o_ChannelMember_partnerImStatusIcon position-absolute bottom-0 end-0 d-flex align-items-center justify-content-center text-light'"
                         classNameObj="{
                             'o-isDeviceSmall': messaging.device.isSmall,
                             'small': !messaging.device.isSmall,
                         }"
-                        partner="member"
+                        partner="channelMember.partner"
                     />
                 </t>
             </div>
-            <span class="o_ChannelMember_name ml-2 flex-column-1 text-truncate o_cursor_pointer" t-on-click="() => channel.onClickMemberName(member)">
-                <t t-esc="member.nameOrDisplayName"/>
+            <span class="o_ChannelMember_name ml-2 flex-column-1 text-truncate o_cursor_pointer" t-on-click="channelMember.onClickName">
```

Comment:
diff unrelated to PR, should be moved to not yet existing channel member view instead